docs/whatsnew/2.0.rst

```diff
@@ -0,0 +1,202 @@
+    >>> height = 10 * u.arcsec
+    >>> bottom_left, top_right = get_rectangle_coordinates(bottom_left, width=width, height=height)
+    >>> bottom_left, top_right
+    (<SkyCoord (HeliographicStonyhurst: obstime=None): (lon, lat, radius) in (deg, deg, km)
+     (0., 0., 695700.)>,
+    <SkyCoord (HeliographicStonyhurst: obstime=None): (lon, lat, radius) in (deg, deg, km)
+     (0.00277778, 0.00277778, 695700.)>)
+
+    # With bottom left of shape (2,)
+    >>> bottom_left_vector = SkyCoord([0 * u.arcsec, 10 * u.arcsec], [0 * u.arcsec, 10 * u.arcsec], frame='heliographic_stonyhurst')
+    >>> bottom_left, top_right = get_rectangle_coordinates(bottom_left_vector)
+    >>> bottom_left, top_right
+    (<SkyCoord (HeliographicStonyhurst: obstime=None): (lon, lat, radius) in (deg, deg, km)
+        (0., 0., 695700.)>,
+    <SkyCoord (HeliographicStonyhurst: obstime=None): (lon, lat, radius) in (deg, deg, km)
+        (0.00277778, 0.00277778, 695700.)>)
+
+    # With bottom left as a BaseCoordinateFrame instance, width and height
+    >>> bottom_left =  HeliographicStonyhurst(0 * u.arcsec, 0 * u.arcsec)
+    >>> width = 10 * u.arcsec
+    >>> height = 10 * u.arcsec
+    >>> bottom_left, top_right = get_rectangle_coordinates(bottom_left, width=width, height=height)
+    >>> bottom_left, top_right
+    (<HeliographicStonyhurst Coordinate (obstime=None): (lon, lat, radius) in (deg, deg, km)
+        (0., 0., 695700.)>,
+    <HeliographicStonyhurst Coordinate (obstime=None): (lon, lat, radius) in (deg, deg, km)
+        (0.00277778, 0.00277778, 695700.)>)
+
+Download behind proxies
+=======================
+
+With the release of parfive 1.1, sunpy has been patched now to able to pass proxy information to `aiohttp.ClientSession`.
+
+* Proxy URL is read from the environment variables `HTTP_PROXY` or `HTTPS_PROXY`, depending on the protocol of the `url` passed.
+* Proxy Authentication `proxy_auth` should be passed as a `aiohttp.BasicAuth` object, explicitly by the user.
+* Proxy Headers `proxy_headers` should be passed as `dict` object, explicitly by the user.
+
+For example if you use a bash terminal:
+
+.. code-block:: bash
+
+    $ HTTP_PROXY=http://user:password@proxyserver.com:3128
+    $ HTTPS_PROXY=https://user:password@proxyserver.com:3128
+    $ export HTTP_PROXY
+    $ export HTTPS_PROXY
+
+these will be used to enable downloads through a proxy.
+
+Citation update
+===============
+
+We had a new paper accepted in, The Astrophysical Journal and you can find the bibtex for it by::
+
+    >>> import sunpy
+    >>> sunpy.__citation__
+
+or `accessing the website directly <https://iopscience.iop.org/article/10.3847/1538-4357/ab4f7a>`__.
+
+aiaprep is now deprecated
+=========================
+
+With the release of the new `aiapy <https://aiapy.readthedocs.io>`__ package, `sunpy.instr.aia.aiaprep` will be removed in version 2.1.
+You can use the `register` function in the aiapy package to convert AIA images to level 1.5.
+
+Search Attributes
```

Comment:
I propose we stick this at the top. It's probably our largest feature this release.